tools/src/icon4py/tools/py2fgen/wrappers/dycore_wrapper.py

```diff
@@ -464,34 +467,4 @@ def grid_init(
     vertical_size: gtx.int32,
     limited_area: bool,
 ):
-    # todo: write this logic into template.py
-    if isinstance(limited_area, int):
-        limited_area = bool(limited_area)
-
-    global_grid_params = GlobalGridParams(level=global_level, root=global_root)
-
-    dycore_wrapper_state["grid"] = wrapper_common.construct_icon_grid(
-        grid_id="icon_grid",
-        global_grid_params=global_grid_params,
-        num_vertices=num_vertices,
-        num_cells=num_cells,
-        num_edges=num_edges,
-        vertical_size=vertical_size,
-        limited_area=limited_area,
-        on_gpu=True if device == "GPU" else False,
-        cell_starts=cell_starts.ndarray,
-        cell_ends=cell_ends.ndarray,
-        vertex_starts=vertex_starts.ndarray,
-        vertex_ends=vertex_ends.ndarray,
-        edge_starts=edge_starts.ndarray,
-        edge_ends=edge_ends.ndarray,
-        c2e=c2e.ndarray,
-        e2c=e2c.ndarray,
-        c2e2c=c2e2c.ndarray,
-        e2c2e=e2c2e.ndarray,
-        e2v=e2v.ndarray,
-        v2e=v2e.ndarray,
-        v2c=v2c.ndarray,
-        e2c2v=e2c2v.ndarray,
-        c2v=c2v.ndarray,
-    )
+    dycore_wrapper_state["grid"] = diffusion_wrapper.diffusion_wrapper_state["grid"]
```

Comment:
this is a temporary workaround to just use the diffusion grid, which is the same except it can already do multinode.